configs/trip_mode_choice_coeffs.csv

```diff
@@ -115,3 +115,5 @@ joint_ride_hail_ASC_tnc_shared,0,0,0,0,0,0,0,0,0,0
 #,,,,,,,,,,
 c_walk_express_penalty,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt,10.00 * c_ivt
 adjust_tnc_shared,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt,30 * c_ivt
+cordon_toll,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt,650 * c_ivt
+income_low_100_discount,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt,300 * c_ivt
```

Comment:
I don't quite understand this part (not to say I think it's wrong!) -- if we're including the cordon toll as part of the "value toll" that already exists, wouldn't that mean we don't need to change the utility functions at all?

Reply:
Here I'm just defining the new coefficients to be used in the 'value toll' expression. Just defining the toll is 650 cents and the discount is 300 cents